src/app/components/Figure/index.stories.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
+import Figure from './index';
+
+const image = {
+  alt:
+    'Map of the UK displaying Syrian refugees and asylum seekers per 10000 population. Ranges from 0 to 17.',
+  src:
+    'https://ichef.bbci.co.uk/news/640/cpsprodpb/439A/production/_100960371_syrians_and_asylum_v2-nc.png',
+};
+
+const caption = 'This is a caption!';
```

Comment:
Can we add a story without a caption to show the variation of the figure component

Reply:
This is addressed in https://github.com/bbc/simorgh/pull/296/files#diff-29479ec42fe7d690788103a2bc221ba7R16